tests/_config/bootstrap.php

```diff
@@ -18,7 +18,11 @@
  */
 $root = dirname(dirname(__DIR__)) . DIRECTORY_SEPARATOR;
 
-loadEnvironment($root);
+loadIni();
+loadAutoloader($root);
+loadFolders();
+loadDefined();
+//loadEnvironment($root);
```

Comment:
Do we still need this in comments. We can clean it up if you ask me. We can always look back in git. Goes for this whole file